src/collections/Form/Form.js

```diff
@@ -102,7 +102,7 @@ class Form extends Component {
 
     // Heads up! Third party libs can pass own data as first argument, we need to check that it has preventDefault()
     // method.
-    if (!action) _.invoke(e, 'preventDefault')
+    if (action === undefined) _.invoke(e, 'preventDefault')
```

Comment:
I think that `false` and `null` are also blocking values. Let's update condition and test assertions

Reply:
Ok sure, I was thinking `false` and `null` would be incorrectly typed here.